velox/exec/ExchangeSource.h

```diff
@@ -94,7 +101,17 @@ class ExchangeSource : public std::enable_shared_from_this<ExchangeSource> {
   // Returns runtime statistics. ExchangeSource is expected to report
   // background CPU time by including a runtime metric named
   // ExchangeClient::kBackgroundCpuTimeMs.
-  virtual folly::F14FastMap<std::string, int64_t> stats() const = 0;
+  virtual folly::F14FastMap<std::string, int64_t> stats() const {
+    VELOX_UNREACHABLE();
+  }
+
+  /// Returns runtime statistics. ExchangeSource is expected to report
+  /// Specify units of individual counters in ExchangeSource.
+  /// for an example: 'totalBytes ：count: 9, sum: 11.17GB, max: 1.39GB,
+  /// min:  1.16GB'
+  virtual folly::F14FastMap<std::string, RuntimeMetric> metrics() const {
```

Comment:
To maintain naming consistency, it is better to name this newly added interface `stats()` too, which is an overload of the old interface.
In my observation, `"stats"` is widely used in Velox, and `"metrics"` is used almost nowhere.

CC @mbasmanova

Reply:
@lingbin You have a good point, but I don't think it is possible to have two method names with same name that differ only in the return type.

Reply:
My mistake, let's rename it in the future after switching to the new interface, thanks.

Reply:
@lingbin Renaming is not straightforward as we need to maintain backwards compatibility. The steps would be:

```
- add stats() method; mark metrics as deprecated
- update Prestissimo to use stats()
- remove metrics() method
```

3 PRs in total. 2 in Velox repo. 1 in PrestoDB repo.